src/WebJobs.Script/Description/FunctionDescriptor.cs

```diff
@@ -43,7 +43,7 @@ internal FunctionDescriptor()
 
         public IFunctionInvoker Invoker { get; internal set; }
 
-        public FunctionMetadata Metadata { get; internal set; }
+        public virtual FunctionMetadata Metadata { get; internal set; }
```

Comment:
>virtual [](start = 15, length = 7)

IF this is just a poco property, why is it virtual?

Reply:
It's for unit tests (Mock.Setup)